docs/guidelines/local-development.rst

```diff
@@ -42,27 +48,68 @@ repository locally by executing:
 
   $ git clone https://github.com/cekit/example-common-module.git ~/repos/example-common-module
 
-Then we will create ``override.yaml`` file next to the ``image.yaml``. Override.yaml should look like this:
+Then we will create ``override.yaml`` file next to the ``image.yaml`` with following content:
 
 .. code-block:: yaml
 
   modules:
     repositories:
-      - path: "/home/user/repo/cct_module"
+      - name: common
+        path: /home/user/repo/cct_module
 
-Now we can build the image using overridden module by executing:
+Now we can build the image with Docker using overridden module by executing:
 
 .. code-block:: bash
 
-  $ cekit generate --overrides-file overrides.yaml
+  $ cekit build --overrides-file overrides.yaml docker
```

Comment:
Spelling of overrides is different to above

Reply:
Nice catch!